source/summary.rst

```diff
@@ -40,41 +40,40 @@ funded through the EU Horizon 2020 programme.
 Attack model and terminology
 ++++++++++++++++++++++++++++
 
-We consider a *network adversary* that can read, modify, and create
-network messages.
-Examples of such an adversary are an ISP, an e-mail provider, an AS,
-or an eavesdropper on a wireless network.
-The goal of the adversary is to i) read the content of messages, ii)
-impersonate peers -- communication partners, and iii) to learn who communicates
-with whom.
-To achieve these goals,
-an active adversary might try, for example,
-to perform a machine-in-the-middle attack on the key exchange protocol
-between peers.
-
-To enable secure key-exchange and key-verification between peers,
-we assume that peers have access to a *out-of-band*
-communication channel that cannot be observed or manipulated by the adversary.
-More concretely we expect them to be able
-to transfer a small amount of data via a QR-code confidentially.
-
-Targeted attacks on end devices or the out-of-band channels
-can break our assumptions
-and therefore the security properties of the protocols described.
-In particular
-the ability to observe QR-codes in the scan process
-(for example through CCTV or by getting access to print outs)
-will allow impersonation attacks.
-Additional measures can
-relax the security requirements for the *out-of-band* channel
-to also work under a threat of observation.
+We consider
+
+- *Peer* devices that use the network or *first channel* for transporting messages
+  and for key-exchange in order to establish end-to-end encryption.
+
+- A *network adversary* that can read, modify, and create
+  network messages on the *first channel*.
+  Examples of such an adversary are an ISP, an e-mail provider, an AS,
+  or an eavesdropper on a wireless network.
+  The goal of the adversary is to i) read the content of messages,
+  and to ii) impersonate *peer* devices.
+
+We assume that
+
+- All peers are honest and do not collaborate with the network adversary.
+
+- A Peer (Alice) can send a single QR-code sized *invite code*
+  in a *second channel* to another peer (Bob).
+  The attacker can not observe or modify the invite code.
```

Comment:
Again here, I subjectively preferred the old formulation "a *out-of-band* communication channel that cannot be observed or manipulated by the adversary.", because it made clear that this is a new word that's newly introduced here ("a *second* communication channel that cannot be observed or manipulated by the adversary." would be just as fine) - and again, I'm probably biased.

Reply:
i re-introduced out-of-band channel.